back-end/service.py

```diff
@@ -221,7 +224,7 @@ def deleteKeysOnServer(token):
                 log('error', response.text)
                 raise ServerError(
                     "You'll have to manually delete the keys from the server.",
-                    401)
+                    HTTPStatus(419))
```

Comment:
I think we should stick to the HTTP standard, 419 is not a part of it. We can just return `401` here.

Reply:
Changed and pushed.